scripts/rollup.config.js

```diff
@@ -0,0 +1,97 @@
+import babel from 'rollup-plugin-babel';
+import commonjs from 'rollup-plugin-commonjs';
+import json from 'rollup-plugin-json';
+import resolve from 'rollup-plugin-node-resolve';
+import uglify from 'rollup-plugin-uglify';
+import pkg from '../package.json';
+
+const date = new Date();
+
+const banner =
+`/**
```

Comment:
Not really a problem here (I actually prefer it like this), but we are going to have to do some work to get it to work with sass in the generator.

Reply:
Yep, that's why it has a separate tool that works across the board in the generator.